lib/templateConfigValidator.js

```diff
@@ -65,9 +65,17 @@ function isRequiredParamProvided(configParams, templateParams) {
  * @param {Object} configParams Parameters specified in template configuration
  */
 function provideAdvice(wrongParams, configParams) {
+  const suggestion = (p) => {
+    const sortInt = (a, b) => {
+      return a[0] - b[0];
+    };
```

Comment:
Just a suggestion, you might like this syntax:
```suggestion
    const sortInt = (a, b) => a[0] - b[0];
```